### ISSUE TITLE
Comments for Vue, Font Awesome, and Facebook/Twitter Icons

### ISSUE THREAD

**Issue by jonjensen** [EndPointCorp/end-point-blog#1443]
Comments for https://www.endpoint.com/blog/2018/07/12/vue-fontawesome-facebook-twitter

To enter a comment:

1. Log in to GitHub
2. Leave a comment on this issue.

**Comment by tabana** [comment]
THANK YOU!

**Comment by rodmi714** [comment]
Thank you very much!

**Comment by GilTads** [comment]
Great job! Thank you so much!

**Comment by PandaSobao** [comment]
Hi All,

**TL;DR**
Found out you can also do:
`<font-awesome-icon :icon="['fab', 'twitter']" />`

**Details**
The following:
`<font-awesome-icon icon="spinner" />`

Is just an alias for:
`<font-awesome-icon :icon="['fas', 'spinner']" />`

Which is just another alias for what this article detailed:
`<font-awesome-icon :icon="{ prefix: 'fab', iconName: 'twitter' }"/>`

**References**
Can find related documentation on this in the vue-fontawesome repo [here](https://github.com/FortAwesome/vue-fontawesome#the-icon-property)

Special thanks to this Medium [comment ](https://medium.com/@JoyrexJ9/hey-nice-article-but-theres-a-far-better-way-to-use-the-brand-icons-than-your-workaround-e24140223a60) for putting this Vue novice on the right track.

**PS**
A lot of little details for integrating icons, but aesthetics, right? 

Wonder if there is a simple way to register the global component with the `'fab'` brand prefix in order to have a brand specific component that looks something like:
`<font-awesome-brand icon="twitter" />`

Or if you register the component with a shorter name:
`<fab icon="twitter" />`

Goals.
